mockData/generateMockData.js

```diff
@@ -53,7 +55,7 @@ loremIpsum = [
 
 const getNameForAllEntries = (words, numberOfEntries) => {
   let allWords = [];
```

Comment:
I think the style guidelines suggest we use const for reference types, and const unless we are changing a reference/have to change a reference - https://github.com/airbnb/javascript#references